### PR TITLE
Bump task describe command to v1beta1

### PR DESCRIPTION
**Pull request by piyush-garg**
This will move the task describe command to use v1beta1
resource and fix the issue of describe command/
not working

When there is online v1alpha1 resource available
it retrives v1alpha1 and convert to v1beta1
and render template

It involves some refactoring in action package
for better usability

Add testing for both v1beta1 and v1alpha1

Related to #817

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes
```
Move task describe to v1beta1 and fix the output
```

### CODE REVIEW

File: pkg/cmd/task/describe.go [hunk 1]
```diff
@@ -42,45 +43,42 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .Task.Name }}
 
 {{decorate "inputresources" ""}}{{decorate "underline bold" "Input Resources\n"}}
 
-{{- if not .Task.Spec.Inputs }}
+{{- if not .Task.Spec.Resources }}
```
Comment (vdemeester) [inline pkg/cmd/task/describe.go:46]: So we won't display `Inputs.Resources` on a v1alpha1 cluster with 0.10.x right ?
I think we should… but this is gonna make the template ever more complex :sweat_smile:

Comment (pradeepitm12) [reply pkg/cmd/task/describe.go:46]: Yes, it makes sense to me.

Comment (danielhelfand) [reply pkg/cmd/task/describe.go:46]: A lot of the logic could be taken out of the templates in functions to keep it simpler.

Comment (piyush-garg) [reply pkg/cmd/task/describe.go:46]: Fixed to work with 0.10.x

File: pkg/task/task.go [hunk 2]
```diff
@@ -65,17 +67,52 @@ func List(c *cli.Clients, opts metav1.ListOptions, ns string) (*v1beta1.TaskList
...
+// It will fetch the resource in v1beta1 struct format
+func GetV1beta1(c *cli.Clients, taskname string, opts metav1.GetOptions, ns string) (*v1beta1.Task, error) {
+	unstructuredT, err := actions.Get(taskGroupResource, c, taskname, ns, opts)
 	if err != nil {
 		return nil, err
 	}
 
 	var task *v1beta1.Task
 	if err := runtime.DefaultUnstructuredConverter.FromUnstructured(unstructuredT.UnstructuredContent(), &task); err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to get task from %s namespace \n", ns)
```
Comment (pradeepitm12) [inline pkg/task/task.go:101]: nit: nice to use lower case for error

File: pkg/taskrun/list/list.go [hunk 3]
```diff
@@ -18,7 +18,8 @@ import (
 	"fmt"
 	"os"
 
-	traction "github.com/tektoncd/cli/pkg/actions/list"
+	"github.com/tektoncd/cli/pkg/actions"
+
```
Comment (pradeepitm12) [inline pkg/taskrun/list/list.go:22]: can remove extra line

File: pkg/task/task.go (hunk 2 above)
Comment (piyush-garg) [reply pkg/task/task.go]: fixed

File: pkg/taskrun/list/list.go (hunk 3 above)
Comment (piyush-garg) [reply pkg/taskrun/list/list.go]: fixed

**Pull request merged**